ssr/src/page/icpump/mod.rs

```diff
@@ -31,6 +32,21 @@ pub fn TokenListing(
                     src=details.logo
                     alt=details.token_name.clone()
                 />
+                {move || {
+                    if details.is_nsfw {
```

Comment:
If the extra `<div></div>` is not intentional or doesnt have any significiant eg css as dependency etc, Use `<Show when=condition/>` instead of this it will minimize the need of <div></div> in the else block

example:
```jsx
            <Show when=move || authorized().map(|(a, _)| a).unwrap_or_default()>
                <YoutubeUploadInner canisters=cans_s.get_value() url=url_s.get_value() />
            </Show>
```